seq2simulate/run_make_proviral_data.py

```diff
@@ -407,8 +450,8 @@ def _parse_sam_line(read_id, sam_file, paired_end=False):
                  ] 
     else:
         seq_id = sam_file[read_id][0][RNAME] 
-        read_start = int(sam_file[read_id][0][POS])
-        read_end = read_start+len(sam_file[read_id][0][SEQ])-1
+        read_start = int(sam_file[read_id][0][POS]) - 1 # convert to 0-based
+        read_end = read_start+len(sam_file[read_id][0][SEQ]) - 1
```

Comment:
Should this be -2? The previous code was `1_based_start + len - 1` now this is `0_based_start - 1` so that's one less item